docs/configuration.txt

```diff
@@ -534,6 +545,38 @@ Blobs
   By default blobs will be stored under the same path as normal data.
   A relative path value is interpreted as relative to ``CRATE_HOME``.
 
+Repositories
+------------
+
+Repositories are used to :ref:`backup-restore` a Crate cluster.
+The following settings are used to configure some types of repositories globally.
+
+**repositories.url.allowed_urls**
+  | *Runtime:* ``no``
+
+  Whitelist for urls that can be used with the :ref:`ref-create-repository-types-url`
+  repository type. Wildcards are supported in the host, path, query and fragment parts.
+
+  Supported protocols are configured in
+  :ref:`repositories.url.supported_protocols <conf-repositories-url-supported-protocols>`.
+  Only ``http``, ``https`` and ``ftp`` URLs need to be listed here for usage in
+  :ref:`ref-create-repository-types-url` repsoitories.
+  
+  ``file`` urls listed here must be prefixed with an entry configured in
+  :ref:`path.repo <conf-path-repo>`.
+
+.. _conf-repositories-url-supported-protocols:
+
+**repositories.url.supported_protocols**
+  | *Default:* ``http``, ``https``, ``ftp``, ``file`` and ``jar``
```

Comment:
either be sure what 'jar' means and that it works or remove it here and change the defaults